docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Set default values for required env vars
+check=( DB_HOST DB_PORT DB_NAME DB_USER DB_PASSWORD DB_SCHEME WALTZ_FROM_EMAIL WALTZ_BASE_URL CHANGELOG_FILE )
+for x in "${check[@]}"
+do
+  if [ -z "$x" ]; then 
```

Comment:
#TODO not working

Reply:
@jgiannuzzi @pavlovic-ivan guys would you know how to handle this condition?
The idea i had is to check if the env variable is empty or not provided, if yes then I'm setting each value with switch below

Reply:
@ljubon changing the condition to `if` [ -z "${!x}" ]` should do the trick :)

Reply:
you're a lifesaver! 🥇 
after some time, I also found new way to print it out `if [[ -z $(printenv $x) ]]`
as I also want to catch `null` i would prefer this one, but feel free to provide your opinion 🙏

Reply:
Another thing to note (and it's a different approach entirely) is that bash offers default values with the `${ENV:-default}` notation.

So you _could_ write something like this instead of the whole loop:
```
export DB_HOST=${DB_HOST:-"postgres"}
export DB_PORT=${DB_PORT:-"5432"}
...
```